content/guides/tooling/sdks/salesforce.md

```diff
@@ -0,0 +1,36 @@
+---
+rank: 4
+related_endpoints: []
+related_guides:
+ - authentication/jwt
+ - authentication/oauth2
+related_pages:
+ - sdks-and-tools
+required_guides: []
+related_resources: []
+alias_paths: []
```

Comment:
Please make sure to check for all the redirect URLs to add. In this case /docs/box-for-salesforce-developer-toolkit